curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/time-calculator.md

```diff
@@ -45,7 +45,7 @@ add_time("6:30 PM", "205:12")
 Do not import any Python libraries. Assume that the start times are valid times. The minutes in the duration time will be a whole number less than 60, but the hour can be any whole number.
 
 # --hints--
-Calling `add_time("3:30 PM", "2:12")` should return `5:42 PM`.
+Calling `add_time("3:30 PM", "2:12")` should return `'5:42 PM'`.
```

Comment:
```suggestion
Calling `add_time('3:30 PM', '2:12')` should return `'5:42 PM'`.
```

can you make this change to single quote to the whole file just as you are here?

Reply:
done ⚡